pkg/controller/controller_instance.go

```diff
@@ -1539,9 +1600,13 @@ func (c *controller) processProvisionFailure(instance *v1beta1.ServiceInstance,
 	// requeue this resource.
 	var err error
 	if shouldMitigateOrphan {
-		readyCond := newServiceInstanceReadyCondition(v1beta1.ConditionFalse, startingInstanceOrphanMitigationReason, startingInstanceOrphanMitigationMessage)
-		c.recorder.Event(instance, corev1.EventTypeWarning, readyCond.Reason, readyCond.Message)
-		setServiceInstanceCondition(instance, v1beta1.ServiceInstanceConditionReady, readyCond.Status, readyCond.Reason, readyCond.Message)
+		reason := startingInstanceOrphanMitigationReason
```

Comment:
The reason and message should be more informative than this. It should come from the failed or ready condition. If we are only going to use a generic reason/message, then there is no point in using a condition at all.